Gemfile

```diff
@@ -8,6 +8,8 @@ gem 'coffee-rails', '~> 4.0.0'
 gem 'jquery-rails'
 gem 'jbuilder', '~> 2.0'
 gem 'bcrypt', '~> 3.1.7'
+gem 'nokogiri'
+gem 'mini_portile2'
```

Comment:
Hey, we're locked on these versions, so please remove gemfile stuff from this PR